### DISCUSSION TITLE
Github Developers, please explain the point of "for you" feature.

### DISCUSSION THREAD

**Discussion by innocentius** [community/community#16353] [Other Feature Feedback, Questions, & Ideas] [answered]
As the question, what is the POINT of this feature? 

Github is not Facebook, not Twitter. We come to Github not to socialize, but to work on coding projects. We need concentration, instead of more distractions. If we want to socialize we would already go to other bigger social media platforms. When we are working, we would not want to randomly wandering off to some "recommended" repo and ask "I wonder what's this."

Well, let's assume some of us want YOU to decide what we want to see. That's fine, but at least have the option to opt out of your stupid beta functions completely! Any github repos have a history of the repo and rollback function, can we have these too? Please don't tell me that I can just ignore "For you" by going to the "Following" tab. Fact is, we don't like that either! 

Get it together, please.

**Comment by sgbaird** [comment]
Maybe someone will make a Chrome extension similar to [Tweak New Twitter](https://chrome.google.com/webstore/detail/tweak-new-twitter/kpmjjdhbcfebfjgdnpjagcndoelnidfj?hl=en) that will hide the feed 🤨

I've come across a number of repositories that I'm really glad I saw because of the Following / For You feeds; however, recently it's just felt like a lot of noise (e.g. https://github.com/orgs/github-community/discussions/13144 and a `conda-forge` bot)

**Reply by innocentius** [reply]
Oh, of course people can use extensions like Adblock to kick the whole thing off the table, sure. In fact I'm doing that right now. 

What I'm suggesting is a turn off switch by the developers at least. Because we can block it doesn't mean we can let it slide, because who knows what's gonna happen next.

**Comment by jfullstackdev** [comment, marked as answer]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
